static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -1832,7 +1832,7 @@ var CStudioForms =
               if (acnDraftContent) {
                 unlockBeforeCancel(path);
               } else {
-                if (!form.readOnly && path && path.indexOf('.xml') != -1) {
+                if (!form.readOnly && path && path.indexOf('.xml') != -1 && !me.config.isInclude) {
```

Comment:
Confirm tested with embedded, shared, parent and nested levels of embedded and shared.

Reply:
Confirmed tested with embedded, shared and multiple levels of embedded and shared